ingestion-data/production/collections/nlcd-new-urbanization.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "nlcd-new-urbanization-v2",
```

Comment:
This id and title aren't the same as the one in the dataset-config

Reply:
Thanks for catching this! I had temporarily created an additional collection while testing new vs old COGs. I'll fix the metadata in this PR